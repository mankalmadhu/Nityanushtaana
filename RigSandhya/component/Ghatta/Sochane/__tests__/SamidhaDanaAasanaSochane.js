import React from 'react';
import ಸಮಿಧಾದಾನಆಸನಸೂಚನೆ from '../SamidhaDanaAasanaSochane';
import Renderer from 'react-test-renderer';


jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');

test("ಸಮಿಧಾದಾನಆಸನಸೂಚನೆ" , () => {
  expect(Renderer.create(
    <ಸಮಿಧಾದಾನಆಸನಸೂಚನೆ/>
   ).toJSON()).toMatchSnapshot();

});