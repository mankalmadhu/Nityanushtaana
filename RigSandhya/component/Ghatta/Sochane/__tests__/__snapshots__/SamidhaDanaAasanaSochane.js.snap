// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಸಮಿಧಾದಾನಆಸನಸೂಚನೆ 1`] = `
<ವಿಷಯ
  ವಿನ್ಯಾಸ="ಶೀರ್ಷಿಕೆ"
>
  (ಅಗ್ನಿಯನ್ನು ಪೂರ್ವ/ಉತ್ತರ ಅಭಿಮುಖವಾಗಿ ಇರಿಸಿಕೊ೦ಡು, ಅಗ್ನಿಗೆ ಅಭಿಮುಖವಾಗಿ ಕುಳಿತುಕೊಳ್ಳುವುದು)
</ವಿಷಯ>
`;
